Apps/Playground/node_modules/@babylonjs/react-native/ios/EngineViewManager.mm

```diff
@@ -0,0 +1,72 @@
+#import "BabylonNativeInterop.h"
+
+#import <React/RCTViewManager.h>
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import <MetalKit/MetalKit.h>
+
+@interface EngineView : MTKView
+@end
+
+@implementation EngineView {
+    RCTBridge* bridge;
+    NSRunLoop* runLoop;
+    BOOL initialized;
+}
+
+- (instancetype)init:(RCTBridge*)_bridge runLoop:(NSRunLoop*)_runLoop {
+    if (self = [super initWithFrame:CGRectZero device:MTLCreateSystemDefaultDevice()]) {
+        bridge = _bridge;
+        runLoop = _runLoop;
+        initialized = false;
+        
+        super.translatesAutoresizingMaskIntoConstraints = false;
+        super.colorPixelFormat = MTLPixelFormatBGRA8Unorm_sRGB;
+        super.depthStencilPixelFormat = MTLPixelFormatDepth32Float;
+    }
+    return self;
+}
+
+- (void)setBounds:(CGRect)bounds {
```

Comment:
Might use viewDidLayout instead?

Reply:
As far as I know that only exists on a `ViewController`, which is not used in React Native (you use `UIView`s directly).